src/sentry/static/sentry/app/stores/memberListStore.jsx

```diff
@@ -46,6 +58,23 @@ const MemberListStore = Reflux.createStore({
   getAll() {
     return this.items;
   },
+
+  onSetActiveOrganization(org) {
+    if (!org) {
+      this.org = null;
+      this.me = null;
+    } else if (this.org !== org.id) {
+      this.org = org.id;
+
+      const endpoint = `/organizations/${org.slug}/members/me/`;
+      this.api.requestPromise(endpoint, {method: 'GET'}).then(member => {
+        if (member && member.user) {
+          this.me = member;
+          this.trigger(this.items, 'me');
```

Comment:
When you refactor this and the store listens to the "fetch me success" action, you'll probably want to trigger with `this.items` and `this.me` -- I don't think any of our store listeners actually use the 2nd argument, so we can replace the string with `this.me`.